circuits/cpp/barretenberg/cpp/src/barretenberg/ecc/curves/secp256r1/secp256r1.cpp

```diff
@@ -4,7 +4,9 @@ namespace secp256r1 {
 namespace {
 
 constexpr size_t max_num_generators = 1 << 10;
+// NOLINTNEXTLINE TODO(@zac-williamson) get rid of need for these static variables in Pedersen refactor!
```

Comment:
same comment as above about issue/PR for pedersen refactor

Reply:
done